Tools/CISupport/ews-build/factories.py

```diff
@@ -338,6 +337,22 @@ def __init__(self, platform, configuration=None, architectures=None, additionalA
         self.addStep(AddReviewerToChangeLog())
         self.addStep(ValidateCommitMessage())
         self.addStep(ValidateChangeLogAndReviewer())
+
+
+class MergeQueueFactory(MergeQueueFactoryBase):
+    def __init__(self, platform, **kwargs):
+        super(MergeQueueFactory, self).__init__()
+
+        self.addStep(ValidateChange(verifyMergeQueue=True, verifyNoDraftForMergeQueue=True))
+        self.addStep(Canonicalize())
+        self.addStep(PushCommitToWebKitRepo())
+        self.addStep(SetBuildSummary())
+
+
+class UnsafeMergeQueueFactory(MergeQueueFactoryBase):
```

Comment:
Both `MergeQueueFactory` and `UnsafeMergeQueueFactory` looks exactly the same. Is this intentional for now (since MergeQueue doesn't build and test currently)?
If they are same currently, we can just move even these steps to base class.

Reply:
Intentional for now, `MergeQueueFactory` will be adding tests very soon, as in "will put up the PR later today" soon